.travis.yml

```diff
@@ -32,7 +32,11 @@ install:
   - |
     if [[ -z ${INTEGRATION} ]]; then
       if ! rustup component add rustfmt; then
-        cargo install --git https://github.com/rust-lang/rustfmt --bin rustfmt
+        TARGET=$(rustc -Vv | awk '/host/{print $2}')
+        NIGHTLY=$(curl -s "https://rust-lang.github.io/rustup-components-history/${TARGET}/rustfmt")
+        curl -sSL "https://static.rust-lang.org/dist/${NIGHTLY}/rustfmt-nightly-${TARGET}.tar.xz" | \
```

Comment:
wouldn't it be easier to just install rustfmt via rustup with `--toolchain $NIGHTLY`?

Reply:
Maybe but that would need to install total NIGHTLY toolchain . Also the fmt test
expects rustfmt of the current nigthly.

Reply:
Makes sense. Why shouldn't we keep the `cargo install` method though?

Reply:
That would need about 10 minutes to build rustfmt. Sometimes the build failed because rustfmt doesn't rustup.
I favor of using precompiled version which still work like previous nightly with rustfmt.